lib/clearance/sign_in_guard.rb

```diff
@@ -16,10 +16,10 @@ module Clearance
   #
   #     # in config/initializers/clearance.rb
   #     Clearance.configure do |config|
-  #       config.sign_in_guards = [ConfirmationGuard]
+  #       config.sign_in_guards = ["ConfirmationGuard"]
   #     end
   #
-  #     # in lib/guards/confirmation_guard.rb
+  #     # in app/guards/confirmation_guard.rb (or another auto-loaded path)
```

Comment:
```suggestion
  #     # in app/guards/confirmation_guard.rb
```
I think just updating the path here would be enough to indicate what we want.